lively.ide/tests/js/import-helper-test.js

```diff
@@ -42,8 +43,12 @@ describe('import helper - injection command', function () {
 
   this.timeout(6000);
 
+  before(async () => {
+    await module('lively.morphic').revive();
+  });
+
```

Comment:
Why do we need this explicitly here? It seems to be the only place where this is needed? :O

Reply:
No, this covers up a bug in the freezer, that I forgot to fix.